### PR TITLE
Add new test case for Poker

### PR DESCRIPTION
**Pull request by dstockto**
Ensures ranking based on adding pair values with equal
weight does not erroneously rank different two-pair
hands as equivalent.

Found an exemplar implementation that ranks two pair hands by summing the two face values. This test case will mean an update to the exemplar is needed in order to be correct.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
Great addition!

**Review by SaschaMann — APPROVED** [review]

**Comment by petertseng** [comment]
Would it be valuable to (either instead or in addition) have two hands with different sum? We can have a hand that has 5s and 4s versus a hand that has 6s and 2s. The latter hand has a strictly lower sum but it is higher-ranked.

File: exercises/poker/canonical-data.json [hunk 1]
```diff
@@ -96,6 +96,15 @@
       },
       "expected": ["JD QH JS 8D QC"]
     },
+    {
+      "uuid": "f761e21b-2560-4774-a02a-b3e9366a51ce",
+      "description": "both hands have two pairs that add to the same value, win goes to highest first pair",
```
[the remark this answers is not in the document]
Comment (dstockto) [reply exercises/poker/canonical-data.json:101]: Does "larger" or "higher" work?

**Comment by dstockto** [comment]
I'd be happy to add that case as well - This one was specifically built to make an improvement to an exemplar that calculates this incorrectly. I think the case you mention probably would also fail in that solution too.

File: exercises/poker/canonical-data.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dstockto) [reply exercises/poker/canonical-data.json]: I've updated the wording for that test case - please let me know if that's acceptable or if you've got another idea

**Review by petertseng — APPROVED** [review]

**Pull request merged**